earth_enterprise/src/common/serverdb/serverdbUtilities.cpp

```diff
@@ -0,0 +1,10 @@
+// Copyright 2018 Open GEE Contributors
+#include "serverdbUtilities.h"
+#include <string>
+#include <regex>
+
+std::string ServerdbUtilities::ParseValidJSON(const std::string& javascript_dict) {
```

Comment:
Naming: This function doesn't parse valid JSON.  It applies a string replace to put quotes around object property names.

Correctness: This function would only work for special cases of Java Script code, e.g. where there are no values containing a string of letters followed by a space and a colon.

This is an implementation that requires verifying that no such values may occur in the Java Script code generated by Fusion.  It is also potential source of hard-to-diagnose problems in the future, in case such values come into use.  E.g., how is a developer supposed to know that a Map client will break if a certain sequence of characters occurs in Java Script code?